tests/__tests__/console/ConsoleTable_test.re

```diff
@@ -0,0 +1,428 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */;
+
+open TestFramework;
+open Console.ConsoleTable;
+
+describe("Console.table", ({test}) => {
+  test("Basic cell", ({expect}) => {
+    let actual =
+      Table.createElement(
+        ~columns=[{width: 4}],
+        ~children=[Row.createElement(~children=["Cell"], ())],
+        (),
+      );
+    let expected = "--------\n" ++ "| Cell |\n" ++ "--------";
+    expect.string(actual).toEqual(expected);
+  });
+
+  test("Basic table", ({expect}) => {
+    let actual =
+      Table.createElement(
+        ~columns=[{width: 7}, {width: 4}],
+        ~children=[
+          Row.createElement(~children=["testing", "1234"], ()),
+          Row.createElement(~children=["a", "b"], ()),
+        ],
+        (),
+      );
+    let expected =
```

Comment:
You could use `.toEqualLines` or `String.concat("\n", [...])` here as well, but I think this is also pretty readable